README.md

```diff
@@ -109,77 +96,38 @@ To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
-In order to run this project you need:
-
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
+In order to run this project you need a compatible browser to load the html file
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
-
 ```sh
   cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  git clone https://github.com/yegonkimutai/Portfolio-project
 ```
---->
 
 ### Install
 
 Install this project with:
 
-<!--
-Example command:
-
 ```sh
-  cd my-project
-  gem install
+  cd portfolio-project
+  yarn install
 ```
---->
 
 ### Usage
 
-To run the project, execute the following command:
-
-<!--
-Example command:
-
-```sh
-  rails server
-```
---->
+-I can help in building a product , feature or website through some of my work and experience! Don’t hestiate to contact me.
```

Comment:
[OPTIONAL] Nice using the README template 💯 but this `Usage` section is supposed to give instructions on how to run this project. Make sure that relevant info is added here.